events/events.md

```diff
@@ -172,7 +172,7 @@ featured: true
 | 28 October | WPTP 2012 | San Diego, California |
 | 28 October-1 November | AMTA 2012 | San Diego, California |
```

Comment:
This looks odd.

Probably we should add spaces around dashes in this scenario, and maybe in all scenarios